translations/chs.po

```diff
@@ -701,493 +701,493 @@ msgstr ""
 msgid "Account"
-msgstr ""
+msgstr "账户"
 
 #: ../client/application/game/ui/screens/accounts/screen_signed_out.cpp:19
 msgid "Not signed in"
-msgstr ""
+msgstr "未登录"
 
 #: ../client/application/game/ui/screens/accounts/screen_signed_out.cpp:23
 msgid "Create an account to set your name and sync your data"
-msgstr ""
+msgstr "创建账号以此同步游戏数据"
 
 #: ../client/application/game/ui/screens/accounts/screen_signed_out.cpp:28
 msgid "Register"
-msgstr ""
+msgstr "注册"
 
 #: ../client/application/game/ui/screens/accounts/screen_signed_out.cpp:51
 msgid "Privacy Policy"
-msgstr ""
+msgstr "隐私政策"
 
 #: ../client/application/game/ui/screens/accounts/screen_signed_out.cpp:93
 msgid "You need to be signed-in to upload scores"
-msgstr ""
+msgstr "你需要登录才能上传分数"
 
 #: ../client/application/game/ui/screens/accounts/screen_signed_out.cpp:96
 msgid "You need to be signed-in to play online"
-msgstr ""
+msgstr "你需要登录才能玩在线游戏"
 
 #. Error message shown if to the player when they are trying to view
 #. comments while not signed-in.
 #: ../client/application/game/ui/screens/accounts/screen_signed_out.cpp:101
 msgid "You need to be signed-in to view comments"
-msgstr ""
+msgstr "你需要登录才能查看评论"
 
 #: ../client/application/game/ui/screens/accounts/screen_signed_out.cpp:113
 msgid "Your account"
-msgstr ""
+msgstr "你的账户"
 
 #: ../client/application/game/ui/screens/accounts/screen_register.cpp:22
 msgid "Email is invalid!"
-msgstr ""
+msgstr "邮箱不可用"
 
 #: ../client/application/game/ui/screens/accounts/screen_register.cpp:47
 msgid "Too short. Minimum %s characters."
-msgstr ""
+msgstr "长度过短，需要至少 %s 个字符"
 
 #: ../client/application/game/ui/screens/accounts/screen_register.cpp:54
 msgid "Too long. Maximum %s characters."
-msgstr ""
+msgstr "长度过长，最多输入 %s 个字符"
 
 #: ../client/application/game/ui/screens/accounts/screen_register.cpp:71
 msgid "Register account"
-msgstr ""
+msgstr "注册账号"
 
 #: ../client/application/game/ui/screens/accounts/screen_register.cpp:82
 msgid "Username"
-msgstr ""
+msgstr "用户名"
 
 #: ../client/application/game/ui/screens/accounts/screen_register.cpp:82
 msgid "Confirm password"
-msgstr ""
+msgstr "确认密码"
 
 #: ../client/application/game/ui/screens/accounts/screen_register.cpp:83
 msgid "Confirm email"
-msgstr ""
+msgstr "确认邮箱"
 
 #: ../client/application/game/ui/screens/accounts/screen_register.cpp:132
 msgid "Registering"
-msgstr ""
+msgstr "注册中"
 
 #: ../client/application/game/ui/screens/accounts/screen_register.cpp:139
 msgid "Registration successful."
-msgstr ""
+msgstr "注册成功"
 
 #: ../client/application/game/ui/screens/accounts/screen_register.cpp:141
 msgid "Please check your emails!"
-msgstr ""
+msgstr "请查看你的邮箱"
 
 #: ../client/application/game/ui/screens/accounts/screen_register.cpp:145
 msgid "Failed to register"
-msgstr ""
+msgstr "注册失败"
 
 #: ../client/application/game/ui/screens/accounts/screen_register.cpp:177
 msgid "Username (you can put colors in!):"
-msgstr ""
+msgstr "用户名（你可以改变颜色）："
```

Comment:
Is there meant to be a space before the `:`?

Reply:
hello and thanks for a fast reply，there isn't a space before the '：'，it is probably because the bracket in chinese typewriting  '）' takes up more space than ')' so it seems to be a space before the '：'.